docs/guides/01-getting-started.md

```diff
@@ -1,6 +1,10 @@
 ## Getting started
 
+<<<<<<< HEAD
```

Comment:
There is a little problem here with the merge

Reply:
ah, good catch, sorry